src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -797,9 +801,17 @@ public static CQLStatement.Raw parseStatement(String queryStr) throws SyntaxExce
         }
     }
 
-    private static int measure(Object key, Prepared value)
+    private static int measurePstmnt(Prepared value)
     {
-        return Ints.checkedCast(ObjectSizes.measureDeep(key) + ObjectSizes.measureDeep(value));
+        return Ints.checkedCast(ObjectSizes.measureDeep(value));
+    }
+
+    private static int getSizeForCache(MD5Digest key, Prepared value)
```

Comment:
@bereng I would rename this to `getSizeOfPreparedStatementForCache`

Reply:
sgtm